src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -431,27 +431,31 @@
   </EnumProperty>
 
   <!-- These Native AOT properties should not be visible for class library projects. -->
-  <DynamicEnumProperty Name="Trimming"
-                       DisplayName="Trimming"
-                       Description="Select the desired .NET trimming option for optimizing your application's deployment size and performance."
-                       HelpUrl="https://go.microsoft.com/fwlink/?linkid=2240879"
-                       Category="Publish"
-                       EnumProvider="TrimmingEnumProvider"
-                       MultipleValuesAllowed="False">
-    <DynamicEnumProperty.DataSource>
-      <DataSource HasConfigurationCondition="False"
-                  Persistence="ProjectFileWithInterception"/>
-    </DynamicEnumProperty.DataSource>
-    <DynamicEnumProperty.Metadata>
+  <BoolProperty Name="IsPublishable"
+                DisplayName="Publishing"
+                Description="Allows the &quot;Publish&quot; target to run. This property only affects processes that use *.proj files and &quot;Publish&quot; target, such as &quot;dotnet publish&quot; command. It does not affect publishing in Visual Studio. The default value is true."
```

Comment:
> It does not affect publishing in Visual Studio.

This doesn't seem to be the behavior I've experienced. Even if this is unchecked, I can still Publish, but the Publish doesn't work.

Reply:
I can remove this line from now, although this is copied from the docs: https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#ispublishable